mods/ts/rules/defaults.yaml

```diff
@@ -845,6 +845,8 @@
 		QuantizedFacings: 0
 	RenderVoxels:
 	WithVoxelBody:
+	AffectedBySlope:
+	
```

Comment:
Unnecessary indention.

Reply:
`AffectedBySlope` is a trait to tell if a actor is affected by slope(or ramp). With this trait,  their art asserts will be affect by slope.

Reply:
Sorry again, the "indention"...